integr8ly/delorean
# add set-product-version command

Adds set-product-version command to set the productVersion and operatorVersion in the rhmi_types file to either "CHANGEME" or the discovered operator version.

https://issues.redhat.com/browse/DEL-213

To Verify

1. Checkout this branch and build
2. Run `./delorean ews set-product-version -d cmd/testdata/manageRHMITypes/rhmi_types -p 3scale -v 9.9.9`
3. Verify that the `OperatorVersion3Scale` var has been changed to `9.9.9` and that the `Version3Scale` var has not been changed
4. Run `./delorean ews set-product-version -d cmd/testdata/manageRHMITypes/rhmi_types -p 3scale -v 9.10.0`
5. Verify that the `OperatorVersion3Scale` var has been changed to `9.10.0` and that the `Version3Scale` var has been changed to `CHANGEME`
